ipynb/primary-processing/02-filter_sequences_for_blooms.md

```diff
@@ -97,4 +99,32 @@ Finally, let's do a quick sanity check that we have sequence data, and that the
 >>> assert os.stat(filtered_sequences_100nt).st_size > 0
 ...
 >>> !biom summarize-table -i $observed_blooms_biom | head -n 25
+Num samples: 5
+Num observations: 7
+Total count: 864
+Table density (fraction of non-zero values): 0.400
+
+Counts/sample summary:
+ Min: 1.0
+ Max: 500.0
+ Median: 59.000
+ Mean: 172.800
+ Std. dev.: 197.986
+ Sample Metadata Categories: None provided
+ Observation Metadata Categories: taxonomy
+
+Counts/sample detail:
+10317.000002244: 1.0
+10317.000007118: 1.0
+10317.000005844: 59.0
+10317.000002035: 303.0
+10317.000009111: 500.0
+```
+
+```python
```

Comment:
for instance, these changes should not be part of the pull request. I'm concerned that you may be doing `git add *.md`, if so, I recommend against that as it risks committing unintended changes.